Packs/CommonDashboards/Scripts/UseCaseAdoptionMetrics/UseCaseAdoptionMetrics.yml

```diff
@@ -0,0 +1,22 @@
+commonfields:
+  id: UseCaseAdoptionMetrics
```

Comment:
The ID 'UseCaseAdoptionMetrics' already exists, Please update the file to have a unique

Reply:
this id already exists because it is part of the use case builder pack

Reply:
im worried if i start performing updates it will start breaking the content.

Reply:
Hey @joe-cosgrove,
If the changes were BC they will also fail on BC validation errors- currently I don't see any.
So if the name is already in use, can't you update it to a different name?